.github/workflows/test_build_conda_macos.yml

```diff
@@ -22,8 +22,8 @@ jobs:
       matrix:
         include:
           - repository: pytorch/audio
-            pre-script: packaging/pre_build_script_wheel.sh
-            post-script: packaging/post_build_script_wheel.sh
+            pre-script: packaging/pre_build_script.sh
```

Comment:
You can rebase now that https://github.com/pytorch/test-infra/pull/750 is merged, and these script-name changes shouldn't be needed anymore!